packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -261,7 +261,10 @@ const SettingsSidebarContainer = ({
                                 className="h-[16px] w-[16px] self-start rounded-full stroke-[2px] ltr:mr-2 rtl:ml-2 md:mt-0"
                                 alt={team.name || "Team logo"}
                               />
-                              <p>{team.name}</p>
+                              <p
+                              className="w-1/2 truncate"
+                              >
```

Comment:
Why is this on  3 lines?

Reply:
@zomars probs some Prettier shit, you want it on one?